apps/src/templates/instructions/teacherFeedback/EditableTeacherFeedback.jsx

```diff
@@ -230,11 +231,12 @@ export class EditableTeacherFeedback extends Component {
   }
 
   render() {
-    const {verifiedInstructor, rubric, visible} = this.props;
+    const {rubric, visible} = this.props;
 
-    const {comment, performance, latestFeedback, errorState} = this.state;
+    const {comment, performance, latestFeedback, errorState, allowEditing} =
+      this.state;
 
-    const placeholderWarning = verifiedInstructor
+    const placeholderWarning = allowEditing
       ? i18n.feedbackPlaceholder()
       : i18n.feedbackPlaceholderNonVerified();
```

Comment:
Should this message be changed? It currently says "You must be a verified teacher to leave feedback."

Reply:
I believe this message is still good. We're still requiring verification for feedback in the upper-level courses. `allowEditing` is set to `verifiedInstructor || allowUnverified`, so if it's false it means the teacher is unverified and in a course that requires verification to leave feedback.